src/client/styles/Shared.ts

```diff
@@ -18,3 +18,12 @@ export const topMargin = css`
     margin-top: ${space[6]}px;
   }
 `;
+
+export const disableAutofillBackground = css`
+  :-webkit-autofill,
+  :-webkit-autofill,
+  :-webkit-autofill {
+    background-color: transparent !important;
+    box-shadow: 0 0 0 50px white inset;
+  }
```

Comment:
Am I correct in assuming this draws the "shadow" over the brower default background?

I am not sure why `:-webkit-autofill` is targeted three times.

Reply:
yep - it adds an inner shadow that occludes the default autofill background. seemed like the best suggested solution.
good spot re the three usages, will remove

Reply:
I've changed so the selector targets:
```
:autofill,
:-webkit-autofill {
...
}
```

as recommended here https://developer.mozilla.org/en-US/docs/Web/CSS/:autofill#examples